pkg/keystore/file/key.go

```diff
@@ -83,7 +83,11 @@ func encryptKey(k *ecdsa.PrivateKey, password string, edg keystore.EDG) ([]byte,
 		}
 		addr = a
 	case elliptic.P256():
-		addr = elliptic.Marshal(elliptic.P256(), k.PublicKey.X, k.PublicKey.Y)
+		privKey, err := k.ECDH()
```

Comment:
can we add a very simple unit test that shows that the result of `elliptic.Marshal(elliptic.P256(), k.PublicKey.X, k.PublicKey.Y)` matches exactly the result of your changes

Reply:
here is the [commit](https://github.com/ethersphere/bee/commit/67860265e1b5486039ef153a76567d480be69de3) with added tests... feel free to suggest any improvements